index.ios.js

```diff
@@ -1,27 +1,4 @@
-import React, { Component } from 'react';
-import {
-  AppRegistry,
-  StatusBar,
-  View,
-} from 'react-native';
-import { theme } from './config';
+import { AppRegistry } from 'react-native';
```

Comment:
Oh, I can see that it's hardcoded in the packager :S ... so looks good for me then.